thirdparty/patchmatchnet/eval.py

```diff
@@ -0,0 +1,125 @@
+"""PatchmatchNet evaluation methods (truncated)
+    reference: https://github.com/FangjinhuaWang/PatchmatchNet
+
+"""
+from typing import Tuple
+
+import cv2
+import numpy as np
+
+
+def reproject_with_depth(
```

Comment:
Can you put these changes into Fangjinhua’s upstream repo, as well, after we make a few more edits here?

Reply:
pull request opened in Fangjinhua's repo

Reply:
thanks Ren